docs/source/install.rst

```diff
@@ -0,0 +1,75 @@
+Installing with ``pip``
+=======================
+
+Installing Zipline via ``pip`` is slightly more involved than the average
+Python package.
+
+There are two reasons for the additional complexity:
+
+1. Zipline ships several C extensions that require access to the CPython C API.
+   In order to build the C extensions, ``pip`` needs access to the CPython
+   header files for your Python installation.
+
+2. Zipline depends on `numpy <http://www.numpy.org/>`_, the core library for
+   numerical array computing in Python.  Numpy depends on having the `LAPACK
+   <http://www.netlib.org/lapack>`_ linear algebra routines available.
```

Comment:
Maybe a bit, but I don't think the effort of somehow making these shared would be worth it.  In particular, I don't think there's a good way to do an `include` in the README that GitHub will still render the way we want.